assets/javascripts/modules/validatorFocus.js

```diff
@@ -7,10 +7,26 @@
     }
 
     $('.error-summary a').on('click', function(e) {
+      var $this = $(this);
+      var focusId = $this.attr('data-focuses');
+      if (!focusId)
+      {
+        focusId = ("" + $this.attr("href")).trim();
+        if (focusId.indexOf("#") !== 0)
```

Comment:
That comment was about those two sentences:
> A developer including a link without a hash is it needed? and is there a genuine use case where a url without a hash maybe required?
By backwards compatibility I meant that this code is using 'focusId' for holding 'href' and 'data-focuses' attribute. data-focuses contains a string without hash in it but href can contain anything. If href doesn't starts with hash it's not compatible with previous functionality and should be treated as an external URL.